### PR TITLE
[SPARK-43176][CONNECT][PYTHON][TESTS] Deduplicate imports in Connect Tests

### PR DESCRIPTION
**Pull request by zhengruifeng**
### What changes were proposed in this pull request?
Deduplicate imports in Connect Tests

### Why are the changes needed?
for simplicity

### Does this PR introduce _any_ user-facing change?
No, test-only

### How was this patch tested?
updated unittests

### CODE REVIEW

File: python/pyspark/sql/tests/connect/test_connect_function.py [hunk 1]
```diff
@@ -14,17 +14,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import os
```
Comment (amaliujia) [inline python/pyspark/sql/tests/connect/test_connect_function.py:17]: Seems that this should be kept but not removed?

**Review by HyukjinKwon — APPROVED** [review]
LGTM if tests pass

File: python/pyspark/sql/tests/connect/test_connect_function.py (hunk 1 above)
Comment (zhengruifeng) [reply python/pyspark/sql/tests/connect/test_connect_function.py:17]: Yea, good catch.
I removed it just as per IDEA's suggestiong...

**Comment by HyukjinKwon** [comment]
Merged to master.

**Pull request closed without merging**